# わざとエラー
print('3'+4)